lib/publisher.js

```diff
@@ -46,7 +45,11 @@ Publisher.prototype.publish = function (metadata) {
       rectrack: metadata.get('rectrack'),
       informative: metadata.get('informative'),
       editorDraft: metadata.get('editorDraft'),
-      processRules: metadata.get('processRules')
+      processRules: metadata.get('processRules'),
+      implementationReport: metadata.get('implementationReport'),
+      implementationFeedbackDue: new Moment(metadata.get('implementationFeedbackDue', 'YYYY-MM-DD')
+              .format('YYYY-MM-DD'),
```

Comment:
There's a missing `)` in this expression, and I think the first param `'YYYY-MM-DD'` shouldn't be there?

Apart from that, why
`new Moment(metadata.get('implementationFeedbackDue').format('YYYY-MM-DD')`
?
Isn't
`metadata.get('implementationFeedbackDue')`
already a string with the same format that you need?

Reply:
Oups, I thought I had pushed the missing parenthesis but I didn't. Done now.

I use `Moment` to convert strings like `2018-6-1` to `2018-06-01`. By default, specberus doesn't provide the leading `0`.